.travis.yml

```diff
@@ -1,15 +1,36 @@
 language: php
 
+sudo: false
+
+cache:
+    directories:
+        - $HOME/.composer/cache
+
 php:
     - 5.3
     - 5.4
     - 5.5
     - 5.6
+    - 7.0
     - hhvm
 
-before_script:
-    - "composer require --dev phpspec/phpspec:~2.0.0-RC4 --no-update"
-    - "composer install --no-progress --no-plugins"
+env:
+    global:
+        - TEST_COMMAND="composer test"
+
+matrix:
+    allow_failures:
+        - php: hhvm
+    fast_finish: true
+    include:
+        - php: 5.3
+          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
+
+before_install:
+    - travis_retry composer self-update
+
+install:
+    - travis_retry composer update ${COMPOSER_FLAGS} --prefer-source --no-interaction
```

Comment:
Yes, that's a copy past mistake. Also, prefer-dist does not require Github token anymore.